### PR TITLE
ER-683: Test cards

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-683

### CODE REVIEW

File: sites/all/modules/publizon/includes/publizon.availability.inc [hunk 1]
```diff
@@ -79,13 +109,15 @@ function publizon_availability_items(array $local_ids) {
     $key = $to_check[$isbn];
     $results[$key] = $data;
 
-    // Store in cache
-    $static_cache[$key] = $data;
+    if ($use_cache) {
+      // Store in cache.
+      $static_cache[$key] = $data;
```
Comment (cableman) [inline sites/all/modules/publizon/includes/publizon.availability.inc]: Why not have the static cache out-side the $use_cache. That would be better as this is called more than once on some pages and the static cache is stille checked and is valid for the current user.

File: sites/all/modules/publizon/publizon.module [hunk 2]
```diff
@@ -81,6 +81,15 @@ function publizon_menu() {
     'file' => 'includes/publizon.admin.inc',
   );
 
+  $items['admin/config/ereolen/publizon'] = array(
+    'title' => 'Publizon settings',
+    'description' => 'Settings for the publizon module',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('publizon_admin_settings_form'),
+    'access arguments' => array('configure publizon'),
```
Comment (cableman) [inline sites/all/modules/publizon/publizon.module]: Why not use the existing permission `administer ding provider` ?

File: sites/all/modules/publizon/publizon.module [hunk 3]
```diff
@@ -81,6 +81,15 @@ function publizon_menu() {
...
+    'file' => 'publizon.admin.inc',
```
Comment (cableman) [inline sites/all/modules/publizon/publizon.module]: Why create a new file when `includes/publizon.admin.inc` already has admin forms? Just add it there.

File: sites/all/modules/publizon/includes/publizon.availability.inc (hunk 1 above)
Comment (rimi-itk) [reply sites/all/modules/publizon/includes/publizon.availability.inc]: Fixed!

File: sites/all/modules/publizon/publizon.module (hunk 3 above)
Comment (rimi-itk) [reply sites/all/modules/publizon/publizon.module]: Fixed!

File: sites/all/modules/publizon/publizon.module (hunk 2 above)
Comment (rimi-itk) [reply sites/all/modules/publizon/publizon.module]: Changed.

**Pull request merged**